### PR TITLE
Changes in file

### PR DESCRIPTION
**Pull request by garbich**

### CODE REVIEW

**Comment by garbich** [comment]
Changed in file

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+something
```
Comment (XardasCode) [inline file1.txt:1]: Bad. Change to "Add new something"

Comment (BogdanSadovskiy) [inline file1.txt:1]: lalka

Comment (garbich) [reply file1.txt:1]: shut                                                                                                                         up

Comment (triceratops13) [inline file1.txt:1]: something is bad add "nothing"

**Comment by Dimonch1k** [comment]
Hi pupils

**Pull request merged**